app/client/src/git/sagas/updateGlobalProfileSaga.ts

```
@@ -30,9 +32,14 @@ export default function* updateGlobalProfileSaga(
       yield put(gitConfigActions.updateGlobalProfileSuccess());
       yield put(gitConfigActions.fetchGlobalProfileInit());
     }
-  } catch (error) {
-    yield put(
-      gitConfigActions.updateGlobalProfileError({ error: error as string }),
-    );
+  } catch (e) {
+    if (response && response.responseMeta.error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)